commons/c2cgeoportal_commons/models/main.py

```diff
@@ -1250,9 +1251,7 @@ class LayerCOG(Layer):
                     _(
                         """
             <div class="help-block">
-                <p>Definition of a <code>COG Layer</code>.</p>
-                <p>Note: The layers named <code>cog-defaults</code> contains the values
-                used when we create a new <code>COG layer</code>.</p>
+                <p>URL of the COG</p>
```

Comment:
Here you should remove Literal and help-block:

```
"description": _("URL of the COG"),
```

not sure it works otherwise with the replacement at runtime.